docs/reference/files/manifest-yaml-file.rst

```diff
@@ -0,0 +1,19 @@
+.. _manifest-yaml-file:
+
+
+``manifest.yaml`` file
+======================
+
+    See also: `How to configure
+    Charmcraft <https://juju.is/docs/sdk/charmcraft-config>`_
```

Comment:
This should probably instead go to our internal howto/manage-charms document now, which is the successor to that document.

Reply:
Agreed, I wasn't certain about this link in my initial review.